### ISSUE TITLE
Add SE module to pretrain-model

### ISSUE THREAD

**Issue by buptchan** [moskomule/senet.pytorch#8]
I notice that your implementation on SE-ResNet added SE modules to pretrained ResNet models directly. The ResNet part is pretrained but the the SE modules are randomly initialized. 
So I wonder does this has an impact on the final accuracy ? I would like to hear your answer
about this issue, thank you.  
 : P

**Comment by moskomule** [comment]
Hi, I don't use the pre-trained model (you mean the weights, right?) but use just the Res-Net's structure itself and replace its Res-block with my SE-Res-block.
